184.js

```diff
@@ -0,0 +1,24 @@
+//https://code.mu/ru/javascript/book/prime/inbuilt/math/powering/
+console.log(1, Math.pow(2, 10))
+
+console.log(2, Math.sqrt(245))
+
+let arr3 = [4, 2, 5, 19, 13, 0, 10]
+let sum184 = 0
+for(let elem of arr3){
+    sum184 = sum184 + (elem**3)
+}
+console.log(3, Math.sqrt(sum184))
+
+
+let arr3b = [4, 2, 5, 19, 13, 0, 10]
+let newArr3b = []
+for(let elem of arr3b){
+    newArr3b.push(elem**3)
+}
+let total = newArr3b.reduce(function(a, b){
+    return a + b
+})
+console.log(3, Math.sqrt(total))
```

Comment:
тут `for` не патрэбны
узвадзі ў трэцюю ступень адразу ў `.reduce()`

Reply:
Найдите квадратный корень из суммы кубов его элементов. Для решения воспользуйтесь циклом for.

Reply:
згодзен, тады ў цябе ўсё правільна

калі б не было пра `for`, то вось так можна было рашаць
```suggestion
let arr3b = [4, 2, 5, 19, 13, 0, 10]
let total = arr3b.reduce(function(a, b){
    return a + b**3
})
console.log(3, Math.sqrt(total))
```

Reply:
зачыні гэтую размову, калі азнаёмішся